Yousei.Core/DefaultParameter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Yousei.Shared;
+
+namespace Yousei.Core
+{
+    public class DefaultParameter : IParameter
```

Comment:
Is this just an empty ConstantParameter, when is this used?

Reply:
This always returns the default value for the given type when calling Resolve and is meant to be a singleton default instance for IParameter.